### PR TITLE
[VaultList] CAES-1513: Fix displaying pinned and unpinned vaults

### PR DESCRIPTION
**Pull request by cu4xxi**

### CODE REVIEW

**Review by sa715 — APPROVED** [review]

File: packages/common/sagas/entities/member.js [hunk 1]
```diff
@@ -263,7 +263,9 @@ export function* addMemberToTeamListsBatchSaga({ payload: { teamId, users } }) {
       teamId,
     });
 
-    const membersById = convertMembersToEntity(serverMembers);
+    const membersById = convertMembersToEntity(
+      serverMembers.map(member => ({ ...member, accessGranted: true })),
```
Comment (beshkenadze) [inline packages/common/sagas/entities/member.js:267]: Please move it to the normalizer that's what it for being.

**Review by beshkenadze — CHANGES_REQUESTED** [review]
See a comment

File: packages/common/sagas/entities/member.js (hunk 1 above)
Comment (cu4xxi) [reply packages/common/sagas/entities/member.js]: Done

File: packages/common/normalizers/schemas/member.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const memberSchema = new schema.Entity(
   {
     processStrategy: entity => ({
       ...entity,
+      accessGranted: 'accessGranted' in entity ? entity.accessGranted : true,
```
Comment (beshkenadze) [inline packages/common/normalizers/schemas/member.js:11]: Maybe? accessGranted: !!entry.accessGranted

**Review by beshkenadze — APPROVED** [review]

**Pull request merged**